src/lang/bg-BG.json

```diff
@@ -733,6 +733,7 @@
   "th_biggest_hit": "Biggest Hit",
   "th_lane": "Lane",
   "th_map": "Map",
+  "th_win_lane": "Win",
```

Comment:
Only the en file needs to be updated--we have processes to do translations but they haven't been run in a while

Reply:
If it's not too much trouble--otherwise it may cause conflicts if we ever run the language update script (but TBH we haven't done that in a long time)